test/bugs-reported/bug6.chpl

```diff
@@ -0,0 +1,14 @@
+use Queue;
```

Comment:
I agree that this is because the compiler hasn't found a file named "Queue.chpl".  The solution is to name the file with Queue in it on the compilation line (i.e. add a .compopts file with the additional module file, plus probably --main-module bug6 or something).  I believe this behavior is okay, since there could be multiple files in the same directory which define a module by that name and the compiler wouldn't know which one to choose if it had to look into all files at this directory scope.